src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -131,6 +139,22 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
     ...(isCPS && { pageTitle: `${atiAnalytics.pageTitle} - ${brandName}` }),
   };
 
+  let blocksWithExperimentTopStories = blocks;
+  const topStoriesContent = pageData?.secondaryColumn?.topStories;
+  const shouldEnableExperimentTopStories =
+    enableExperimentTopStories({
+      isAmp,
+      service,
+      pathname,
+    }) && topStoriesContent;
+
+  if (shouldEnableExperimentTopStories) {
```

Comment:
**Suggestion (non blocking):** You could rewrite this block of code using a ternary and that way you can avoid using a `let` for `blocksWithExperimentTopStories`

eg: `const blocksWithExperimentTopStories = shouldEnableExperimentTopStories ? insertExerpimentTopStories() : blocks`

And this can also sit in the helper file you created

Reply:
Nice spot, think this was just from a previous implementation. Updated.